docker-compose.yml

```diff
@@ -126,21 +127,71 @@ services:
       --tracing-endpoint http://jaeger-tracing:14268/api/traces
       --tracing-process-name validator-node
     ports:
-      - "7500:7500"
+      - '7500:7500'
     volumes:
       - type: bind
         source: ./shared/chain-config.yml
         target: /config/chain-config.yml
 
+  lodestar-beacon-node:
+    depends_on:
+      - execution-node
+    build:
+      context: ./lodestar
+      dockerfile: Dockerfile.lodestar
+    environment:
+      EXECUTION_NODE_URL: http://execution-node:8545
+      PROCESS_NAME: lodestar-beacon-node
+    entrypoint: ['run_beacon_node.sh']
+    ports:
+      - '3500:3500'
+      - '13000:13000'
+    volumes:
+      - 'beacon_node_data:/chaindata'
+      - type: bind
+        source: ./shared/chain-config.yml
+        target: /config/chain-config.yml
+      - type: bind
+        source: ./lodestar/run_beacon_node.sh
+        target: /usr/local/bin/run_beacon_node.sh
+
+  lodestar-beacon-node-follower:
+    depends_on:
+      - execution-node-2
+      - lodestar-beacon-node
+    build:
+      context: ./lodestar
+      dockerfile: Dockerfile.lodestar
+    environment:
+      EXECUTION_NODE_URL: http://execution-node-2:8545
+      BEACON_NODE_RPC: http://lodestar-beacon-node:3500
+      PROCESS_NAME: lodestar-beacon-node-follower
+      VERBOSITY: debug
+    entrypoint: run_beacon_node_peer.sh
+    ports:
+      - '3501:3500'
+      - '13001:13000'
+    volumes:
+      - 'beacon_node_follower_data:/chaindata'
+      - type: bind
+        source: ./shared/chain-config.yml
+        target: /config/chain-config.yml
+      - type: bind
+        source: ./lodestar/run_beacon_node.sh
+        target: /usr/local/bin/run_beacon_node.sh
+      - type: bind
+        source: ./lodestar/run_beacon_node_peer.sh
+        target: /usr/local/bin/run_beacon_node_peer.sh
+
   jaeger-tracing:
     image: jaegertracing/all-in-one:1.35
     environment:
       COLLECTOR_ZIPKIN_HTTP_PORT: 9411
     ports:
-      - "5775:5775/udp"
-      - "6831:6831/udp"
-      - "6832:6832/udp"
-      - "5778:5778"
-      - "16686:16686"
-      - "14268:14268"
-      - "9411:9411"
+      - '5775:5775/udp'
+      - '6831:6831/udp'
+      - '6832:6832/udp'
+      - '5778:5778'
+      - '16686:16686'
+      - '14268:14268'
+      - '9411:9411'
```

Comment:
I'm sorry, my editor wants single quotes real bad 🤷‍♂️

Reply:
it's fine :(